pkg/spoke/controllers/manifestwork_controller.go

```diff
@@ -193,6 +193,13 @@ func (m *ManifestWorkController) cleanupResourceOfWork(work *workapiv1.ManifestW
 				"Failed to delete resource %v with key %s/%s: %w",
 				gvr, object.GetNamespace(), object.GetName(), err))
 			continue
+		default:
+			// Return err since we just trigger the delete, and check if resource is not found in
```

Comment:
I'm not entirely clear on why we need to return an error when the delete was successful.  Are you worried about finalization time?

Reply:
yes, i think so.